app/code/core/Mage/Admin/Model/User.php

```diff
@@ -272,7 +272,7 @@ public function deleteFromRole()
     public function roleUserExists()
     {
         $result = $this->_getResource()->roleUserExists($this);
-        return (is_array($result) && count($result) > 0) ? true : false;
+        return (is_array($result) && count($result) > 0);
```

Comment:
Is it better without brackets?

```suggestion
        return is_array($result) && count($result) > 0;
```